front/src/common/Loaders/Loader.tsx

```diff
@@ -74,7 +74,7 @@ export const Loader = ({
   childClass = '',
   className = '',
 }: LoaderProps) => (
-  <div className={`loader ${position} ${className}`}>
+  <div data-testid="modal-loader" className={`loader ${position} ${className}`}>
```

Comment:
It's not a modal loader I think, but only a loader no ?
```suggestion
  <div data-testid="loader" className={`loader ${position} ${className}`}>
```